letsencrypt/models/acme_tiny.py

```diff
@@ -0,0 +1,199 @@
+# flake8: noqa
```

Comment:
[You can `pip install acme-tiny` now](https://github.com/diafygi/acme-tiny/pull/59#issuecomment-214826624), so please remove this file.